docs/content/preview/secure/authentication/host-based-authentication.md

```diff
@@ -97,6 +97,29 @@ hostnossl  database  user  IP-address  netmask  auth-method  [auth-options]
 
```

Comment:
out-of-scope: local, host, hostssl, hostnossl probably belong in a subcategory similar to how auth methods are